pkgs/ffigen/doc/objc_ios_mac_differences.md

```diff
@@ -0,0 +1,102 @@
+# Dealing with iOS/macOS API differences
+
```

Comment:
Consider cross-linking this file with the "Dealing with Objective-C API versioning" file and vice versa. Due to the similarity in the title I can see some people accidentally ending up here even though they are interested in the other one.

Reply:
Good point. Probably better to just combine them in that case.